app/panel/components/BuildingBlocks/DonutGraph.jsx

```diff
@@ -314,23 +314,24 @@ class DonutGraph extends React.Component {
 	 */
 	render() {
 		const { isSmall, totalCount } = this.props;
-		const componentClasses = ClassNames('sub-component', 'donut-graph', {
-			small: isSmall,
-			big: !isSmall,
+		const componentClasses = ClassNames('DonutGraph', {
+			'DonutGraph--big': !isSmall,
+			'DonutGraph--small': isSmall,
 		});
 
+		// TODO Foundation dependency: tooltip
```

Comment:
Is this still a ToDo?  I thought I saw a git commit that added this back.  Not 100% sure, just checking?

Reply:
Leaving as is to help locate Foundation dependencies in case we move forward with removing Foundation in the future.